common/app/common/pages/RatingData.java

```diff
@@ -0,0 +1,24 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class RatingData {
+    private final CmsPage cms;
+
+    public RatingData(final CmsPage cms) {
+        this.cms = cms;
+    }
+
+    public List<SelectableData> getRating() {
+        return asList(
+                new SelectableData("5 Stars", "5", cms.getOrEmpty("ratingFiveStarText"), "", false),
```

Comment:
@trangng but do we have any kind of list we can use to decide what is CMS content and what is not? Besides asking you :P